lnrpc/rpc.proto

```diff
@@ -1283,6 +1283,29 @@ message HTLC {
     uint32 expiration_height = 4;
 }
 
+enum CommitmentType {
+    /**
+    A channel using the legacy commitment format having tweaked to_remote
+    keys.
+    */
+    LEGACY = 0;
```

Comment:
Legacy as a type isn't very scalable. Static remote key will be legacy at some point too.

Reply:
OG? :P

Reply:
GENESIS

Reply:
sticking with legacy 😛